lib/fluent/plugin/exception_detector.rb

```diff
@@ -79,8 +79,7 @@ def self.supported
 
     PYTHON_RULES = [
       rule(:start_state, /^Traceback \(most recent call last\):$/, :python),
-      rule(:python, /^[\t ]+File /, :python_code),
-      rule(:python_code, /[^\t ]/, :python),
+      rule(:python, /^[\t ]+[\S]+.*/, :python),
```

Comment:
See https://github.com/GoogleCloudPlatform/fluent-plugin-detect-exceptions/issues/62#issuecomment-543136795.

Reply:
reverted and added second entrypoint instead